### ISSUE TITLE
503 using fastlane match

### ISSUE THREAD

**Issue by justincpollard** [fastlane/fastlane#26526]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [ ] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
There's a 503 error when trying to reach the fastlane match service. During the normal build process, this error pops up on the `get_push_certificate` step. But also appears by just running `fastlane match`
```
[12:56:29]: Verifying that the certificate and profile are still valid on the Dev Portal...

Looking for related GitHub issues on fastlane/fastlane...


[!] The request could not be completed because:
	<html>
<head><title>503 Service Temporarily Unavailable</title></head>
<body>
<center><h1>503 Service Temporarily Unavailable</h1></center>
<hr><center>Apple</center>
</body>
</html>
```

##### Command executed
`fastlane match`

##### Complete output when running fastlane, including the stack trace and command used
<details>
  <pre>
fastlane match

[✔] 🚀
[12:56:26]: fastlane detected a Gemfile in the current directory
[12:56:26]: However, it seems like you didn't use `bundle exec`
[12:56:26]: To launch fastlane faster, please use
[12:56:26]:
[12:56:26]: $ bundle exec fastlane match
[12:56:26]:
[12:56:26]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
[12:56:27]: Successfully loaded 'REDACTED/fastlane/Matchfile' 📄

+---------------------------------------------------------------------------------------+
|                      Detected Values from './fastlane/Matchfile'                      |
+--------------+------------------------------------------------------------------------+
| git_url      | ssh://git@github.com/REDACTED											|
| storage_mode | git                                                                    |
| type         | appstore                                                               |
+--------------+------------------------------------------------------------------------+

+-----------------------------------------------------------------------------------------------------------------+
|                                            Summary for match 2.224.0                                            |
+----------------------------------------+------------------------------------------------------------------------+
| type                                   | appstore                                                               |
| readonly                               | false                                                                  |
| generate_apple_certs                   | true                                                                   |
| skip_provisioning_profiles             | false                                                                  |
| app_identifier                         | ["REDACTED"]                                                           |
| username                               | REDACTED                                                               |
| team_id                                | REDACTED                                                               |
| storage_mode                           | git                                                                    |
| git_url                                | ssh://git@github.com/REDACTED                                          |
| git_branch                             | master                                                                 |
| shallow_clone                          | false                                                                  |
| clone_branch_directly                  | false                                                                  |
| skip_google_cloud_account_confirmation | false                                                                  |
| s3_skip_encryption                     | false                                                                  |
| gitlab_host                            | https://gitlab.com                                                     |
| keychain_name                          | login.keychain                                                         |
| force                                  | false                                                                  |
| force_for_new_devices                  | false                                                                  |
| include_mac_in_profiles                | false                                                                  |
| include_all_certificates               | false                                                                  |
| force_for_new_certificates             | false                                                                  |
| skip_confirmation                      | false                                                                  |
| safe_remove_certs                      | false                                                                  |
| skip_docs                              | false                                                                  |
| platform                               | ios                                                                    |
| derive_catalyst_app_identifier         | false                                                                  |
| fail_on_name_taken                     | false                                                                  |
| skip_certificate_matching              | false                                                                  |
| skip_set_partition_list                | false                                                                  |
| force_legacy_encryption                | false                                                                  |
| verbose                                | false                                                                  |
+----------------------------------------+------------------------------------------------------------------------+

[12:56:27]: Cloning remote git repo...
[12:56:27]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
[12:56:29]: Checking out branch master...
[12:56:29]: 🔓  Successfully decrypted certificates repo
[12:56:29]: Verifying that the certificate and profile are still valid on the Dev Portal...

Looking for related GitHub issues on fastlane/fastlane...

[!] The request could not be completed because:
	<html>
<head><title>503 Service Temporarily Unavailable</title></head>
<body>
<center><h1>503 Service Temporarily Unavailable</h1></center>
<hr><center>Apple</center>
</body>
</html>
  </pre>
</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 15.0.1                                      |
| Ruby                        | 3.3.5                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.39.5 (Apple Git-154)          |
| Installation Source         | ~/.asdf/installs/ruby/3.3.5/bin/fastlane    |
| Host                        | macOS 15.0.1 (24A348)                       |
| Ruby Lib Dir                | ~/.asdf/installs/ruby/3.3.5/lib             |
| OpenSSL Version             | OpenSSL 3.3.2 3 Sep 2024                    |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 16.0                                        |
| Swift Version               | 6.0                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
default_platform(:ios)

platform :ios do
    desc "AppCenter Release"
    lane :build_qa do
      changelog = prompt(
        text: "Changelog: ",
        multi_line_end_keyword: "END"
      )

      cocoapods(
        clean_install: true,
        verbose: true
      )

      increment_build_number(
        build_number: sh("date", "+'%Y%m%d.%H%M'"),
        xcodeproj: "REDACTED"
      )

      commit_version_bump(
        message: "REDACTED",
        xcodeproj: "REDACTED"
      )

      get_push_certificate(
        app_identifier: "REDACTED"
      )

      match(type: "adhoc",
        app_identifier: "REDACTED",
        force_for_new_devices: true
      )

      build_app(workspace: "REDACTED", scheme: "REDACTED", include_bitcode: false, clean: true)
      
      appcenter_upload(
        api_token: "REDACTED",
        owner_name: "REDACTED",
        owner_type: "organization", # Default is user - set to organization for appcenter organizations
        app_name: "REDACTED",
        file: "REDACTED",
        dsym: "REDACTED",
        notify_testers: true, # Set to false if you don't want to notify testers of your new release (default: `false`)
        release_notes: changelog
      )

      clean_build_artifacts           # Delete the local dSYM files
    end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
REDACTED
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.224.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                     | Version | Update-Status |
| -------------------------- | ------- | ------------- |
| fastlane-plugin-sentry     | 1.25.1  | ✅ Up-To-Date  |
| fastlane-plugin-versioning | 0.6.0   | ✅ Up-To-Date  |
| fastlane-plugin-appcenter  | 2.1.2   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| error_highlight                 | 0.6.0        |
| did_you_mean                    | 1.6.3        |
| syntax_suggest                  | 2.0.0        |
| babosa                          | 1.0.4        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| dotenv                          | 2.8.1        |
| emoji_regex                     | 3.2.3        |
| faraday-cookie_jar              | 0.0.7        |
| gh_inspector                    | 1.1.3        |
| google-apis-playcustomapp_v1    | 0.13.0       |
| google-cloud-env                | 1.6.0        |
| naturally                       | 2.2.1        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.5        |
| simctl                          | 1.6.10       |
| terminal-notifier               | 2.0.0        |
| terminal-table                  | 3.0.2        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| word_wrap                       | 1.0.0        |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| xcpretty-travis-formatter       | 1.0.1        |
| set                             | 1.1.0        |
| public_suffix                   | 4.0.7        |
| addressable                     | 2.8.7        |
| artifactory                     | 3.0.17       |
| aws-eventstream                 | 1.3.0        |
| aws-sigv4                       | 1.10.0       |
| aws-partitions                  | 1.992.0      |
| jmespath                        | 1.6.2        |
| aws-sdk-core                    | 3.210.0      |
| aws-sdk-kms                     | 1.95.0       |
| aws-sdk-s3                      | 1.169.0      |
| bundler                         | 2.5.22       |
| base64                          | 0.2.0        |
| nkf                             | 0.2.0        |
| rexml                           | 3.3.8        |
| CFPropertyList                  | 3.0.7        |
| excon                           | 0.112.0      |
| ruby2_keywords                  | 0.0.5        |
| faraday-retry                   | 1.0.3        |
| faraday-rack                    | 1.0.0        |
| faraday-patron                  | 1.0.0        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-net_http                | 1.0.2        |
| multipart-post                  | 2.4.1        |
| faraday-multipart               | 1.0.4        |
| faraday-httpclient              | 1.0.1        |
| faraday-excon                   | 1.1.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-em_http                 | 1.0.0        |
| faraday                         | 1.10.4       |
| faraday_middleware              | 1.2.1        |
| domain_name                     | 0.6.20240107 |
| http-cookie                     | 1.0.7        |
| fastimage                       | 2.3.1        |
| httpclient                      | 2.8.3        |
| multi_json                      | 1.15.0       |
| jwt                             | 2.9.3        |
| signet                          | 0.19.0       |
| os                              | 1.1.4        |
| googleauth                      | 1.8.1        |
| mini_mime                       | 1.1.5        |
| retriable                       | 3.1.2        |
| trailblazer-option              | 0.1.2        |
| declarative                     | 0.0.20       |
| uber                            | 0.1.0        |
| representable                   | 3.2.0        |
| google-apis-core                | 0.11.3       |
| google-apis-androidpublisher_v3 | 0.54.0       |
| rake                            | 13.2.1       |
| digest-crc                      | 0.6.5        |
| google-apis-storage_v1          | 0.31.0       |
| google-apis-iamcredentials_v1   | 0.17.0       |
| google-cloud-errors             | 1.4.0        |
| google-cloud-core               | 1.7.1        |
| google-cloud-storage            | 1.47.0       |
| json                            | 2.7.2        |
| mini_magick                     | 4.13.2       |
| optparse                        | 0.5.0        |
| plist                           | 3.7.1        |
| unicode-display_width           | 2.6.0        |
| tty-screen                      | 0.8.2        |
| nanaimo                         | 0.3.0        |
| colored2                        | 3.1.2        |
| claide                          | 1.1.0        |
| atomos                          | 0.1.3        |
| xcodeproj                       | 1.25.1       |
| forwardable                     | 1.3.3        |
| logger                          | 1.6.1        |
| pathname                        | 0.3.0        |
| shellwords                      | 0.2.0        |
| cgi                             | 0.4.1        |
| date                            | 3.3.4        |
| timeout                         | 0.4.1        |
| stringio                        | 3.1.1        |
| securerandom                    | 0.3.1        |
| uri                             | 0.13.1       |
| ipaddr                          | 1.2.6        |
| openssl                         | 3.2.0        |
| digest                          | 3.1.1        |
| io-nonblock                     | 0.3.0        |
| zlib                            | 3.1.1        |
| resolv                          | 0.3.0        |
| io-wait                         | 0.3.1        |
| time                            | 0.3.0        |
| open-uri                        | 0.4.1        |
| mutex_m                         | 0.2.0        |
| net-http                        | 0.4.1        |
| net-protocol                    | 0.2.2        |
| english                         | 0.8.0        |
| erb                             | 4.0.3        |
| strscan                         | 3.0.9        |
| abbrev                          | 0.1.2        |
| io-console                      | 0.7.1        |
| tempfile                        | 0.2.1        |
| delegate                        | 0.3.1        |
| fileutils                       | 1.7.2        |
| tmpdir                          | 0.2.0        |
| singleton                       | 0.2.0        |
| open3                           | 0.2.1        |
| prettyprint                     | 0.2.0        |
| pp                              | 0.5.0        |
| find                            | 0.2.0        |
| ostruct                         | 0.6.0        |
| yaml                            | 0.3.0        |
| psych                           | 5.1.2        |
| fastlane-plugin-sentry          | 1.25.1       |
| fastlane-plugin-versioning      | 0.6.0        |
| fastlane-plugin-appcenter       | 2.1.2        |
| csv                             | 3.2.8        |
</details>

*generated on:* **2024-10-18**
</details>

**Comment by laraferroni** [comment]
See if this update fixes it for you (it hasn't for me yet):

https://github.com/fastlane/fastlane/issues/26368

**Comment by justincpollard** [comment]
@laraferroni thanks, seems fixed by fastlane version 2.225.0. Closing this issue
